tests/providers/google/cloud/operators/test_gcs_to_bigquery.py

```diff
@@ -29,6 +32,15 @@
 TEST_SOURCE_OBJECTS = ['test/objects/*']
 
 
+@pytest.mark.backend("mysql", "postgres")
```

Comment:
System tests should be in the test_gcs_to_bigquery_sysstem.py file

Reply:
In the future, we will want to move them to a separate directory, because they do not test `airflow.providers.google.cloud.operators.gcs_to_bigquery` module, but example dag.